include/jemalloc/internal/rtree_structs.h

```diff
@@ -6,9 +6,26 @@ struct rtree_node_elm_s {
 };
 
 struct rtree_leaf_elm_s {
-	atomic_p_t	le_extent; /* (extent_t *) */
+#if RTREE_NHIB >= LG_CEIL_NSIZES
+	/*
+	 * Single pointer-width field containing all three leaf element fields.
+	 * For example, on a 64-bit x64 system with 48 significant virtual
+	 * memory address bits, the index, extent, and slab fields are packed as
+	 * such:
+	 *
+	 * x: index
+	 * e: extent
+	 * b: slab
+	 * k: lock
+	 *
+	 *   11111111 xxxxxxxx eeeeeeee [...] eeeeeeee eeee00bk
```

Comment:
Don't we only have 3 low bits available? Or are extents always 16B aligned? Also, why making the highest 8 bits 1s instead of 0s?